### PR TITLE
Update set-up-common-area-phones.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/1856
Added important note voicemail.

### CODE REVIEW

File: Teams/set-up-common-area-phones.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ description: "Learn how to set up Common Area Phones for lobbies, reception area
 ---
 
 # Set up the Common Area Phone license for Microsoft Teams
+> [!NOTE]
+> Common Area Phones do not support voicemail.
```
Comment (nenonix) [inline Teams/set-up-common-area-phones.md]:
```suggestion
> Common area phones do not support voicemail.
```

**Comment by scanum** [comment]
@LolaJacobsen Changes look correct, could you please review and merge? Thanks!

**Comment by scanum** [comment]
@LolaJacobsen Hi, any updates on this? Thanks!

**Comment by LolaJacobsen** [comment]
@CarolynRowe - could you review this one please?

**Pull request merged**